src/Transpiler.zig

```diff
@@ -882,8 +908,24 @@ fn visitCXXConstructorDecl(self: *Self, value: *const json.Value) !void {
                 try self.out.print("{s}: {s}", .{ arg, z_type });
             } else if (mem.eql(u8, arg_kind, "FormatAttr")) {
                 // varidatic function with the same properties as printf
-            } else {
-                self.nodes_visited -= 1;
+                handled_inner_nodes.items[i] = true;
+                self.nodes_visited += 1;
+            } else if (mem.eql(u8, arg_kind, "CXXCtorInitializer")) {
+                // constructor initializer for a single variable, not interesting here
+                handled_inner_nodes.items[i] = true;
+                self.nodes_visited += 1;
+                // try self.visitCXXCtorInitializer(item);
+            } else if (mem.eql(u8, arg_kind, "CompoundStmt")) {
+                handled_inner_nodes.items[i] = true;
+                self.nodes_visited += 1;
+                // try self.visitCompoundStmt(item);
```

Comment:
why did you removed the `else { log.err("unhandled `{s}` in ctor `{s}`", .{ arg_kind, parent }); }` in favor of this `handled_inner_nodes` ?

Reply:
Maybe there was a smarter way to do this :) But I added the docs outputting, and then I didn't want to get errors about missing the comment nodes.

Yeah I think now it would've been better to simply check for those nodes in the second for loop too and ignore them. Can do that if you want!

Reply:
Yes, the simpler the better

Reply:
Yeah, updated this branch now, will fix the rest tomorrow!